pipeline/extract/document.py

```diff
@@ -123,7 +127,30 @@ def save_text(self, text_filepath: Path):
         with open(text_filepath, "wt") as f:
             f.write(self.to_string())
 
+    @classmethod
+    def from_dict(cls, json_dict: dict) -> "Document":
+        """Load a document from a dictionary"""
+        for page in json_dict["pages"]:
```

Comment:
Fixed bug where the text blocks within pages were not conforming ot the Doc>Page>TB format.